server/datastore/mysql/microsoft_mdm.go

```diff
@@ -64,21 +42,35 @@ func (ds *Datastore) MDMWindowsGetEnrolledDeviceWithDeviceID(ctx context.Context
 	return &winMDMDevice, nil
 }
 
-// MDMWindowsInsertEnrolledDevice inserts a new MDMWindowsEnrolledDevice in the database
+// MDMWindowsInsertEnrolledDevice inserts a new MDMWindowsEnrolledDevice in the
+// database.
 func (ds *Datastore) MDMWindowsInsertEnrolledDevice(ctx context.Context, device *fleet.MDMWindowsEnrolledDevice) error {
 	stmt := `
 		INSERT INTO mdm_windows_enrollments (
-		mdm_device_id,
-		mdm_hardware_id,
-		device_state,
-		device_type,
-		device_name,
-		enroll_type,
-		enroll_user_id,
-		enroll_proto_version,
-		enroll_client_version,
-		not_in_oobe,
-		host_uuid ) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+			mdm_device_id,
+			mdm_hardware_id,
+			device_state,
+			device_type,
+			device_name,
+			enroll_type,
+			enroll_user_id,
+			enroll_proto_version,
+			enroll_client_version,
+			not_in_oobe,
+      host_uuid)
+		VALUES
+			(?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+		ON DUPLICATE KEY UPDATE
+			mdm_device_id         = VALUES(mdm_device_id),
+			device_state          = VALUES(device_state),
+			device_type           = VALUES(device_type),
+			device_name           = VALUES(device_name),
+			enroll_type           = VALUES(enroll_type),
+			enroll_user_id        = VALUES(enroll_user_id),
+			enroll_proto_version  = VALUES(enroll_proto_version),
+			enroll_client_version = VALUES(enroll_client_version),
+			not_in_oobe           = VALUES(not_in_oobe),
+      host_uuid             = VALUES(host_uuid)
```

Comment:
Nit: formatting

Reply:
thank you!! this is what I get for using the GH UI to resolve conflicts.